hana/lib/HANAService.js

```diff
@@ -1295,8 +1295,10 @@ SELECT ${mixing} FROM JSON_TABLE(SRC.JSON, '$' COLUMNS(${extraction})) AS NEW LE
   }
 
   async _getProcedureMetadata(name, schema) {
-    const query = `SELECT PARAMETER_NAME FROM SYS.PROCEDURE_PARAMETERS WHERE SCHEMA_NAME = ${schema?.toUpperCase?.() === 'SYS' ? `'SYS'` : 'CURRENT_SCHEMA'
-      } AND PROCEDURE_NAME = '${name}' AND PARAMETER_TYPE IN ('OUT', 'INOUT') ORDER BY POSITION`
+    const sqlString = this.class.CQN2SQL.prototype.string
+    name = typeof name === 'string' ? sqlString(name) : `'${name}'`
+    schema = typeof schema === 'string' ? sqlString(schema) : 'CURRENT_SCHEMA'
```

Comment:
What does `sqlString` do?
Might be related to the failing test

Reply:
`sqlString` is coming from `cqn2sql` which implements how to inline strings into a SQL statement ([code](https://github.com/cap-js/cds-dbs/blob/641c3b97d0571b8b8b7f44e038ffc34ac17c49e5/db-service/lib/cqn2sql.js#L1013-L1015)).

Reply:
The test was failing, because it was testing functionalities not implemented and the original regexp would return `<schema>.` as the schema name.